lib/IRGen/IRGenModule.cpp

```diff
@@ -1646,6 +1646,7 @@ void IRGenModule::addLinkLibraries() {
                                               getSwiftModule()->getName().str(),
                                               hasStaticCxx,
                                               hasStaticCxxStdlib,
+                                              Context.LangOpts.CXXStdlib,
```

Comment:
Unnecessary change?

Reply:
We do use this value in `registerCxxInteropLibraries` to determine which C++ stdlib be should link against.

Reply:
Hmm, weird, it showed up a whitespace only change 😕